### PR TITLE
HtmlEditor: Fix history buttons state after call clearHistory method(T1133799)

### PR DESCRIPTION
**Pull request by alexanderPolosatov**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/api.tests.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ import { isObject } from 'core/utils/type';
 const { test, module: testModule } = QUnit;
 
 const TOOLBAR_FORMAT_WIDGET_CLASS = 'dx-htmleditor-toolbar-format';
+const DISABLED_STATE_CLASS = 'dx-state-disabled';
+const DX_BUTTON_CLASS = 'dx-button';
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/api.tests.js]: I don't think there can be some "not dx" button class)
```suggestion
const BUTTON_CLASS = 'dx-button';
```

File: testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/api.tests.js [hunk 2]
```diff
@@ -254,19 +256,40 @@ testModule('API', moduleConfig, () => {
         assert.strictEqual(this.instance.option('value'), '<p>cbaTest 1</p><p>Test 2</p><p>Test 3</p>', 'redo operation');
     });
 
-    test('clearHistory', function(assert) {
+    test('value should not changed when "undo" called after call "clearHistory"', function(assert) {
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/api.tests.js]:
```suggestion
    test('value should not change on undo after "clearHistory" call', function(assert) {
```

File: testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/api.tests.js [hunk 3]
```diff
@@ -254,19 +256,40 @@ testModule('API', moduleConfig, () => {
...
+        this.options.toolbar = { items: ['redo'] };
+        this.createEditor();
+        this.instance.insertText(0, 'a');
+        this.clock.tick(1000);
+        const $redoButton = $('#htmlEditor').find(`.${DX_BUTTON_CLASS}`);
+
         this.instance.undo();
+        this.instance.clearHistory();
 
-        assert.strictEqual(this.instance.option('value'), '<p>cbaTest 1</p><p>Test 2</p><p>Test 3</p>', 'history is empty');
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/api.tests.js:269]: hm, we kept such a test for undo, but removed for redo. Let's keep it as well?

Comment (alexanderPolosatov) [reply testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/api.tests.js:269]: I did not remove step with redo but I will add test for redo call

**Review by ksercs — APPROVED** [review]

**Review by Zedwag — APPROVED** [review]

**Pull request merged**